components/canvas/Cargo.toml

```diff
@@ -38,12 +38,12 @@ servo_arc = { path = "../servo_arc" }
 servo_config = { path = "../config" }
 sparkle = { workspace = true }
 style = { path = "../style" }
-style_traits = { path = "../style_traits" }
+style_traits = { workspace = true }
```

Comment:
This now references the  'style_traits'  in workspace which has the 'servo' feature enabled. Does this have any negative implications?  My understanding is features are `unioned` together during crate compilation, so compile time shouldn't be impacted. But wouldn't this allow `canvas` to now accidentally refer to implementation details exposed using `servo` feature ?

Should we specify `features = [ "servo" ]` wherever it is needed (`components/servo/Cargo.toml`)  and not enabled it in the workspace?

Reply:
You're correct on all the details, yet I don't think it's an issue that canvas accesses parts of `style_traits` that use the `servo` feature. In general, all of servo should have access to the `servo` version of style and it's better that it's more widely available to all crates now.

Reply:
Hmm...of the 12 (not sure if I miscounted) consumers of `style_traits` only 2 need the `servo` feature, so it seemed to me that the other crates were designed to be independent of those features. But I don't have much insight into what `style_trait` represents or how it ought to be consumed, so I'll defer to your judgement.

Reply:
`servo` isn't really an optional feature for `style` and `style_traits`. You really need to use either `servo` or `gecko`. In this case, it doesn't make as much sense to support compilation without one of the two.